frontend/tests/resources/write_onecallchain/0.c

```diff
@@ -24,8 +24,13 @@ int x = 1;
 int y = x*x + 3;
 int z = y*x + 12345;
 int w = z*z + x - y;
-void (*frontend_f)(void) = function_6;
-frontend_f();
+function_5();
+label76:;
```

Comment:
nit: can we drop the semicolon after a label? we can postpone that for another commit if it's going to change a lot of other tests.

Reply:
It's needed as labels can only be followed by statements and not declarations.